pkg/overlay/cache.go

```diff
@@ -107,51 +107,48 @@ func (o *Cache) Bootstrap(ctx context.Context) error {
 			return err
 		}
 	}
-	// called after kademlia is bootstrapped
-	// needs to take RoutingTable and start to persist it into the cache
-	// take bootstrap node
-	// get their route table
-	// loop through nodes in RT and get THEIR route table
-	// keep going forever and ever
-
-	// Other Possibilities: Randomly generate node ID's to ask for?
-
-	_, err = o.DHT.GetRoutingTable(ctx)
 
 	return err
 }
 
-// Refresh walks the network looking for new nodes and pings existing nodes to eliminate stale addresses
+// Refresh will run as often as configured and updates the cache db
```

Comment:
running as often as configured seems up to the caller of Refresh, right? doesn't need to be in this comment. perhaps this comment should say "Refresh runs once"

Reply:
Updated